content/pt-br/docs/tasks/configure-pod-container/configure-gmsa.md

```diff
@@ -0,0 +1,356 @@
+
+## Crie recursos de especificação de GMSA
+
+Com o `GMSACredentialSpec CRD` instalado (como descrito anteriormente), 
+recursos customizados contendo recursos de especificação de credenciais GMSA podem 
+ser configurados. A especificação de credencial GMSA não contém segredos nem dados 
+sensíveis. É informação que o contêiner runtime pode usar para descrever a apropriada 
+GMSA de um contêiner para o Windows. Especificações de credenciais GMSA podem 
+ser geradas em formato YAML com o utilitário [PowerShell script](https://github.com/kubernetes-sigs/windows-gmsa/tree/master/scripts/GenerateCredentialSpecResource.ps1).
+
+A seguir são os passos para gerar a especificação de credencial GMSA YAML 
+manualmente, em formato JSON e então convertê-la:
+
+1. Importar a `CredentialSpec` [módulo](https://github.com/MicrosoftDocs/Virtualization-Documentation/blob/live/windows-server-container-tools/ServiceAccounts/CredentialSpec.psm1): 
+`ipmo CredentialSpec.psm1`
+
+1. Crie a especificação da credencial em formato JSON usando `New-CredentialSpec`. 
+Para criar a especificação da credencial GMSA nomeada WebApp1, 
+execute `New-CredentialSpec -Name WebApp1 -AccountName WebApp1 -Domain $(Get-ADDomain -Current LocalComputer)`
+
+1. Use `Get-CredentialSpec` para mostrar o caminho do arquivo JSON.
+
+1. Converta o arquivo `credspec` de JSON para o formato YAML e aplique os campos 
+de cabeçalho necessários `apiVersion`, `kind`, `metadata` e `credspec` para fazer 
+o recurso customizado `GMSACredentialSpec` que pode ser configurado no Kubernetes.
+
+A configuração YAML a seguir descreve as especificações de credencial GMSA nomeada 
+`gmsa-WebApp1`:
+
+```yaml
+apiVersion: windows.k8s.io/v1
+kind: GMSACredentialSpec
+metadata:
+  name: gmsa-WebApp1      #Este é um nome arbitrário, mas será usado como referência
+credspec:
+  ActiveDirectoryConfig:
+    GroupManagedServiceAccounts:
+    - Name: WebApp1       #Nome de usuário da conta GMSA
+      Scope: CONTOSO      #Nome de Domínio NETBIOS
+    - Name: WebApp1       #Nome de usuário da conta GMSA
+      Scope: contoso.com  #Nome de domínio DNS
+  CmsPlugins:
+  - ActiveDirectory
+  DomainJoinConfig:
+    DnsName: contoso.com        #Nome de domínio DNS
+    DnsTreeName: contoso.com    #Nome de domínio DNS Root
+    Guid: 244818ae-87ac-4fcd-92ec-e79e5252348a  #GUID
+    MachineAccountName: WebApp1 #Nome de usuário da conta GMSA
+    NetBiosName: CONTOSO        #Nome de domínio NETBIOS
+    Sid: S-1-5-21-2126449477-2524075714-3094792973 #SID da GMSA
+```
+
+O recurso de especificação de credencial acima deve ser salvo como 
+`gmsa-Webapp1-credspec.yaml` e aplicado no cluster usando: 
+`kubectl apply -f gmsa-Webapp1-credspec.yml`
+
+## Configure a `cluster role` para habilitar RBAC nas especificações 
+de credenciais GMSA específicas
+
+Uma `cluster role` precisa ser definida para cada recurso de especificação 
+de credencial GMSA. Isto autoriza a `usar` um recurso GMSA específico, 
```

Comment:
"`use`" should not be translated here, as it is a specific verb in the cluster role spec. Suggestion: "autoriza o verbo `use` em um recurso GMSA específico".